README.md

```diff
@@ -37,4 +37,44 @@ The configuration file follows a simple structure.
         ...
     ]
 }
-```
\ No newline at end of file
+```
+
+### Adding variable type
+
+You can specify the type of a variable by extending its name with the type in the query as such: ```variableName_variableType```. The underscore ```_``` here is crutial to make a clear distinction between name and type. 
```

Comment:
- Explain why someone would want to add the type.
- Give an example.

Reply:
This is resolved in [commit 259d097](https://github.com/SolidLabResearch/generic-data-viewer/pull/6/commits/259d0979b5cee6cb7d7348032e3ab3404323d93a)

Reply:
> either a URL or a literal value

This statement is incorrect. You can also get a blank node https://www.w3.org/TR/rdf12-concepts/#section-blank-nodes

Reply:
Thank you, I have adjusted this in [commit 16ad860](https://github.com/SolidLabResearch/generic-data-viewer/pull/6/commits/16ad8604d233a32e95dd04e969fb48197f2bb0d1)